README.md

```diff
@@ -262,6 +261,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 
 #### Security
 
+- [Snyk Test Action](https://github.com/snyk/actions)
```

Comment:
This should actually be moved to "Collection of Actions" since it is a collection of actions. Would you mind moving it to the bottom of that list?

Reply:
I don't see how it makes sense to be inside the collection of actions. How is it different than the "scan for docker images" on the security list? which is what this Snyk Actions does as well (both docker images and apps dependencies).

Reply:
update: I moved the snyk entry to the bottom of the security list

Reply:
You are correct, I briefly saw there was a list of actions for each of their supported platforms but a deeper dive shows it's just "Run Snyk to check for vulnerabilities".

Reply:
Sure thing, thanks!